config/default/manager_webhook_patch.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        ports:
+        - containerPort: 9443
+          name: webhook-server
+          protocol: TCP
+        volumeMounts:
+        - mountPath: /tmp/k8s-webhook-server/serving-certs
```

Comment:
attempt to run the PR locally with `make run` and on the cluster by `make deploy` fails with:
```
E0518 04:52:31.676884       1 leaderelection.go:330] error retrieving resource lock osbuild-operator-system/bfdcaedc.osbuilder.project-flotta.io: Get "https://10.96.0.1:443/apis/coordination.k8s.io/v1/namespaces/osbuild-operator-system/leases/bfdcaedc.osbuilder.project-flotta.io": context canceled
1.652849551676954e+09   ERROR   setup   problem running manager {"error": "open /tmp/k8s-webhook-server/serving-certs/tls.crt: no such file or directory"}
main.main
        /workspace/main.go:105
runtime.main
        /usr/local/go/src/runtime/proc.go:255
```

Reply:
@masayag Is this issue specific to this PR? Sounds like it's from the previous commit

Reply:
yes, the error includes a missing file which is referred to by this PR.
I don't get that error without it.

Reply:
after taking the update PR, the error isn't reproduced.